### ISSUE TITLE
Issue with entity values

### ISSUE THREAD

**Issue by gkf01** [golastmile/rasa_nlu#106]
Hi,

I am trying to map synonyms and misspellings to predefined values (using spacy + sklearn). In demo-rasa.json, I add the following (plus several other possibilities): 
{
        "text": "show me tchineez restaurants", 
        "intent": "\"restaurant_search\"", 
        "entities": [
          {
            "start": 8, 
            "end": 16, 
            "value": "\"chinese\"", 
            "entity": "cuisine"
          }
        ]
}
After training, when I send "show me tchineez restaurants" to parse, I get "tchineez" as the entity value instead of the "chinese".

**Comment by amn41** [comment]
Indeed this isn't currently implemented. See `entity normalization` [here in the roadmap](http://rasa-nlu.readthedocs.io/en/latest/roadmap.html). 

The simplest way to do this would just be to create a list of aliases in your code & reconcile them later. Support for doing this natively (& more generally) is coming, but I can't tell you exactly when.

**Comment by amn41** [comment]
My suggestion for how to implement this is to build up a map when we read through the training data. All entities are then mapped by their "value". So the map would look like : 
```
{ 
  "tchineez" : "chinese",
  "chinese" : "chinese",
  "china" : "chinese"
  ...etc
}
```
built up from entities like 
```
{  
   "text":"show me tchineez restaurants",
   "intent":""   restaurant_search"",
   "entities":[  
      {  
         "start":8,
         "end":16,
         "value":"chinese",
         "entity":"cuisine"
      }
   ]
}
```

We would persist this in the model dir, as `index.json` or something
